src/lib.rs

```diff
@@ -118,10 +193,21 @@ impl RouteGroup {
     }
 }
 
+/// A clearable connection to a [Mailbox]. Addressed in [PostOffice] by [Address].
 struct Route {
```

Comment:
I see, so a Route is a managed `Sender`, to which you can subscribe for closures as well, and a mailbox is an owned non-cloneable `Receiver` :+1: